test/callbacks.jl

```diff
@@ -0,0 +1,113 @@
+using StatsFuns
+using ForwardDiff: Dual
+
```

Comment:
Why do you call this file `callbacks.jl`? Isn't it mainly to test that generic types exemplified by `Dual`s work?